code/src/cljs/sixsq/nuvla/ui/pages/edges_detail/views.cljs

```diff
@@ -2140,21 +2140,20 @@
 
 (defn EdgeDetails
   [uuid]
+  (refresh-nuvlaedge-data uuid)
```

Comment:
By removing the 2 form component the refresh will be called each time the nb-status is being updated server side. 
I think the bug is in the fact that we are making a deref on nb-status at component mount time.